persistence-modules/spring-data-jpa-4/src/main/resources/application.properties

```diff
@@ -2,4 +2,12 @@ spring.jpa.show-sql=true
 #MySql
 #spring.datasource.url=jdbc:mysql://localhost:3306/baeldung
 #spring.datasource.username=baeldung
-#spring.datasource.password=baeldung
\ No newline at end of file
+#spring.datasource.password=baeldung
+#spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+#spring.jpa.database-platform=org.hibernate.dialect.MySQL57Dialect
```

Comment:
Out of interest, would it make more sense to switch to H2? then the user can run the examples without having to standup a MySql server

Reply:
I've continued to use MySql because it was previously used in the configuration, but since it was commented I've changed it now to H2